sensor/PressureAdaptor.java

```diff
@@ -0,0 +1,49 @@
+package sensor;
+
+
+import java.awt.Color;
+
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.border.TitledBorder;
+
+public class PressureAdaptor implements ReadSensor {
+	private PressureSensor ps = new PressureSensor();
+	private double value;
+	private String report;
+	
+	@Override
+	public String readValueStatus() {
+		value = ps.readValue();
+		report = ps.getReport();
+		return value + " --> " + report;
+		
+	}
+
+	@Override
+	public JPanel getPanel() {

```

Comment:
PressureAdaptor.getPanel(), RadiationAdaptor.getPanel(), and TemperatureAdaptor.getPanel() looks similar. I recommend add an abstract class between adaptor class and interface to avoid duplicate code.